go.sum

```diff
@@ -251,6 +251,7 @@ github.com/globalsign/mgo v0.0.0-20180905125535-1ca0a4f7cbcb/go.mod h1:xkRDCp4j0
 github.com/globalsign/mgo v0.0.0-20181015135952-eeefdecb41b8 h1:DujepqpGd1hyOd7aW59XpK7Qymp8iy83xq74fLr21is=
 github.com/globalsign/mgo v0.0.0-20181015135952-eeefdecb41b8/go.mod h1:xkRDCp4j0OGD1HRkm4kmhM+pmpv3AKq5SU7GMg4oO/Q=
 github.com/go-acme/lego v2.5.0+incompatible/go.mod h1:yzMNe9CasVUhkquNvti5nAtPmG94USbYxYrZfTkIn0M=
+github.com/go-bindata/go-bindata v3.1.1+incompatible h1:tR4f0e4VTO7LK6B2YWyAoVEzG9ByG1wrXB4TL9+jiYg=
```

Comment:
if `go.mod` does not change, then `go.sum` should not change. Try to revert those changes and see if you can compile and run `make assets` without changes in go.sum.

Reply:
Even on the master, without my changes,` make assets` modifies 

   modified:   go.sum
   modified:   pkg/assets/bindata.go

Reply:
try to revert changes in go.sum. 

pkg/assets/bindata.go will change every time you run `make assets`. Only the first time includes required changes. After that, only timestamps are changed. Not a big deal. But if you change a bit of the template, do not forget to run `make assets` once again.